graphql/resolvers.go

```diff
@@ -254,6 +254,52 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, sortCatego
 	return testPointers, nil
 }
 
+func setSchedule(ctx context.Context, sc data.Connector, taskID string, isActive bool) (*restModel.APITask, error) {
+	usr := route.MustHaveUser(ctx)
+	if err := model.SetActiveState(taskID, usr.Username(), isActive); err != nil {
+		fmt.Println(err)
+		return nil, &gqlerror.Error{
```

Comment:
Why not use InternalServerError? Also I would pass through the error text as well